flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+    "only-arches": [
+        "x86_64"
```

Comment:
why?

Reply:
The program and its modules (*-ncnn-vulkan) are built to x86_64

Reply:
why not building from source?

Reply:
I cannot properly support other architectures.

Reply:
why?

Reply:
My main focus is on x86_64, as that is what is most available to me. It would not be properly maintained or supported if I compiled for anything else.

Reply:
it's sure you are not giving it a chance.